src/core/constants.js

```diff
@@ -240,6 +244,7 @@ export const CATEGORY_COLORS = {
   [ADDON_TYPE_EXTENSION]: 10,
   [ADDON_TYPE_LANG]: 12,
   [ADDON_TYPE_OPENSEARCH]: 12,
+  [ADDON_TYPE_STATIC_THEME]: 12, // TODO: verify this is 12
```

Comment:
I guess `12` is fine because we have to make these themes look like the older/other themes.

Reply:
sounds good to me : )